front/lib/auth.ts

```diff
@@ -49,9 +49,15 @@ export class Authenticator {
    *
    * @param session any NextAuth session
    * @param wId string target workspace id
+   * @param asSuperUser boolean if true, will return an admin role for the user if the user is a
+   * superuser
    * @returns Promise<Authenticator>
    */
-  static async fromSession(session: any, wId: string): Promise<Authenticator> {
+  static async fromSession(
```

Comment:
NIT: could be an object over a list of params 🙇🏻‍♀️ 
(feel free to discard)

Reply:
😬